ncm-metaconfig/src/main/perl/metaconfig.pm

```diff
@@ -44,16 +42,6 @@ sub prepare_action
         }
     }
 
-    if ($srv->{daemon}) {
```

Comment:
i'm sorry, but this is even more of a stretch. this code has to stay in to deal with old templates. we can deprecated it (or even throw a warning/error here, but not just rip it out).